app/(golems)/golems/components/CopyButton.tsx

```
@@ -14,17 +14,37 @@ export default function CopyButton({ text }: { text: string }) {
       type="button"
     >
       {copied ? (
-        <svg className="w-4 h-4" fill="none" viewBox="0 0 24 24" stroke="currentColor" strokeWidth={2}>
-          <path strokeLinecap="round" strokeLinejoin="round" d="M5 13l4 4L19 7" />
+        <svg
+          className="h-4 w-4"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+          strokeWidth={2}
+        >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)